<?xml version="1.0"?>
<launch>

    <!-- LAUNCH INTERFACE -->
    <arg name="hardware_interface" default="PositionJointInterface"/>

    <!-- LAUNCH IMPLEMENTATION -->
    <rosparam command="load" file="$(find iiwa_hw)/config/joint_names.yaml" />
    <!-- addresses /-->
    <param name="interface" value="$(arg hardware_interface)"/>

    <!-- the real hardware interface /-->
    <node name="iiwa_hw" pkg="iiwa_hw_grl" type="iiwa_hw_grl-bin" respawn="false" output="screen"/>

</launch>
